BUILD

```diff
@@ -54,6 +55,7 @@ BASE_VARIANTS = [
 BASE = {
     "{REGISTRY}/{PROJECT_ID}/base:{COMMIT_SHA}": "//base:base_root_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/base-debian11:{COMMIT_SHA}": "//base:base_root_amd64_debian11",
+    "{REGISTRY}/{PROJECT_ID}/base-debian12:{COMMIT_SHA}": "//base:base_root_amd64_debian12",
```

Comment:
We need to modify the commands for each "static/base/base-nossl" to build the arch variants, the oci images indexes. 

I think it's actually fine to leave this as is for now, and deal with in a followup though.

Reply:
I'm not sure to follow here, isn't it already the case? Example here:
https://github.com/GoogleContainerTools/distroless/blob/43d0e68992a4a9b0b87aaf66c986088205979218/BUILD#L40-L45

Reply:
omg, yeah you're right, I'm a dumb. I misread the build file.